std/node/process.ts

```diff
@@ -38,6 +38,30 @@ export const process = {
   platform,
   version,
   versions,
+  get stderr() {
+    return {
+      ...Deno.stderr,
```

Comment:
Be explicit about the properties you want to expose here, this is very brittle and can break anytime additions or changes are made to Deno.stderr.